src/type.ts

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable */
+import { Database } from 'sql.js'
```

Comment:
As far as I understand, this file is only used in the `pastel-rpc/network-stats` folder. Please move it there.